tests/test_metrics.py

```diff
@@ -116,98 +89,53 @@ def test_get_preds_implicit_negative_sample_user(
     n_items, n_users = test_implicit_predicted_scores_negative_sample_user.shape
     item_ids = np.arange(n_items)
     model.train_loader.negative_sample_type = 'user'
-    model.return_value = test_implicit_predicted_scores_negative_sample_user.view(-1)
-    actual_preds = get_preds(model=model,
-                             user_or_item_ids=item_ids,
-                             n_items_or_users=n_users,
-                             device=device)
+    model.return_value = test_implicit_predicted_scores_negative_sample_user.reshape(12)
```

Comment:
Why `12` here? Is there a way to make this more dynamic so in case we change the fixture in the future, the test doesn't have to also be changed?

Reply:
Unfortunately, `.view(-1)` doesn't work on the transposed tensor like in the test above. I can replace `12` with `torch.numel` tho:
```suggestion
    model.return_value = test_implicit_predicted_scores_negative_sample_user.reshape(
      torch.numel(test_implicit_predicted_scores_negative_sample_user)
    )
```

Reply:
Love this!